.prettierrc

```diff
@@ -1,10 +1,10 @@
-{
-  "printWidth": 80,
-  "tabWidth": 2,
-  "useTabs": false,
-  "semi": true,
-  "singleQuote": true,
-  "trailingComma": "none",
-  "bracketSpacing": true,
-  "jsxBracketSameLine": false
-}
+{

```

Comment:
Just took a quick glance and noticed that the good old line change from Windows are here: `^M`.
Some of the files are this `.prettierrc` and `LICENSE` to name a few.

## How to reproduce
Checkout this branch and then do e.g `git diff origin/master` to see the files changed and you can see the aforementioned files and others.
At least git can show this when running from macOS terminal, not sure if Windows does some hoxpox and hides these 😄